### ISSUE TITLE
GPU memory problem with GN(Group Normalization)

### ISSUE THREAD

**Issue by Ruikun-Li** [sksq96/pytorch-summary#66]
Hi! Thanks for your great work.
I built a U-net model and used torch-summary to check the GPU memory. When I used BN(Batch Normalization), it cost about 1.8G. But when I used GN(Group Normalization), it cost about 2.2G. When the ipunt is bigger, the difference of the GPU memory will also become bigger. But the problem is, pytorch-summary gave the same costs as below:

BN:(using: torch.nn.BatchNorm2d)
![image](https://user-images.githubusercontent.com/31624867/57195058-a8ccc580-6f80-11e9-8fcc-b06d2302a168.png)

GN:(using: torch.nn.GroupNorm)
![image](https://user-images.githubusercontent.com/31624867/57195062-b71ae180-6f80-11e9-8c2c-b7e07f78e44b.png)

That is the only difference between my code.  I can`t figure out why.
